### PR TITLE
Deleted Azkaban config files from repo (#805)

### PR DESCRIPTION
**Pull request by KyleFung**
Currently, the deleted files are being written into Azkaban jars. These
do not belong here, and can cause hidden configurations from being used.

### CODE REVIEW

**Comment by HappyRay** [comment]
Can we keep a copy of the sample configs somewhere so that people can use them as a reference?

File: azkaban-web-server/src/main/resources/conf/azkaban-users.xml [hunk 1]
```diff
@@ -1,7 +0,0 @@
-<azkaban-users>
-	<user username="azkaban" password="azkaban" roles="admin" groups="azkaban" />
-	<user username="metrics" password="metrics" roles="metrics"/>
```
Comment (fsi206914) [inline azkaban-web-server/src/main/resources/conf/azkaban-users.xml]: When Intellij runs, will this file be enabled? Have you evaluated about this?

**Comment by HappyRay** [comment]
What will be logged if the log4j.properties files are removed and there is no log4j configs provided via the command line? I thought these files provide default logging settings for the application.

**Comment by HappyRay** [comment]
Maybe we can keep the config in the solo server as an example and make solo server easier to setup? see #800

File: azkaban-web-server/src/main/resources/conf/azkaban-users.xml (hunk 1 above)
Comment (suvodeep-pyne) [inline azkaban-web-server/src/main/resources/conf/azkaban-users.xml]: @fsi206914 
I didn't quite understand your question. Can you please clarify your concern?

**Comment by suvodeep-pyne** [comment]
@HappyRay I think the solo server config has been left intact for the exact same reason

**Comment by suvodeep-pyne** [comment]
@HappyRay , the default log4j configuration was leading to exceptions while running tests. The problem is that the default configuration will also conflict with external configuration which is difficult to debug because the log4j.properties gets shipped inside the jar.

The solo server ships with a default log4j config

File: azkaban-web-server/src/main/resources/conf/azkaban-users.xml (hunk 1 above)
Comment (fsi206914) [inline azkaban-web-server/src/main/resources/conf/azkaban-users.xml]: @suvodeep-pyne Never mind. I thought some azkaban-user.xml file might be loaded when Intellij runs the main method of az-solo-server.

**Comment by HappyRay** [comment]
@suvodeep-pyne I thought the default log4j config will be replaced by an external config if provided. What exceptions did you run into?

**Comment by KyleFung** [comment]
@HappyRay The external one overrides the default through the start shell script, or through the "conf" option in the command line. I believe that running tests does not invoke either of these, so no overrides will occur.

**Comment by KyleFung** [comment]
Actually it seems that solo-server relies on the config files that we want to delete. We should add a conf directory solely for the use of solo-server.

**Comment by KyleFung** [comment]
Due to some weird mess up with git, I am continuing this work at #814.

**Pull request closed without merging**